src/inputHandler.ts

```diff
@@ -43,11 +47,12 @@ export async function httpQueryRepoHandler (event: APIGatewayProxyEventV2): Prom
       QueueUrl: process.env.REPO_QUEUE ? process.env.REPO_QUEUE : 'error',
     })
     .promise()
+  return response.MessageId
```

Comment:
is there a harm in returning the full response?

Reply:
The types did not match when I installed them. I wasn't sure what you wanted so I picked a random property.